mods/classic-taskbar-background-fix.wh.cpp

```diff
@@ -0,0 +1,368 @@
+// ==WindhawkMod==
+// @id              classic-taskbar-background-fix
+// @name            Classic Taskbar background fix
+// @description     Fixes Taskbar background in classic theme by replacing black background with a classic button face colour
+// @version         1.0
+// @author          Roland Pihlakas
+// @github          https://github.com/levitation
+// @homepage        https://www.simplify.ee/
+// @compilerOptions -luser32 -lgdi32
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// Source code is published under The GNU General Public License v3.0.
+//
+// For bug reports and feature requests, please open an issue here:
+// https://github.com/levitation-opensource/my-windhawk-mods/issues
+//
+// For pull requests, development takes place here:
+// https://github.com/levitation-opensource/my-windhawk-mods/
+
+// ==WindhawkModReadme==
+/*
+# Classic Taskbar background fix
+
+Fixes Taskbar background in classic theme by replacing black background with a classic button face colour.
+
+Before:
+
+![Before](https://raw.githubusercontent.com/levitation-opensource/my-windhawk-mods/main/screenshots/before-classic-taskbar-background-fix.png)
+
+After:
+
+![After](https://raw.githubusercontent.com/levitation-opensource/my-windhawk-mods/main/screenshots/after-classic-taskbar-background-fix.png)
+*/
+// ==/WindhawkModReadme==
+
+// ==WindhawkModSettings==
+/*
+- RepaintDesktopButton: yes

```

Comment:
How about using a boolean here?

Reply:
There is some chance I will add more options in the future. If I add more options in the future, would it be compatible with users who have already a boolean setting in registry?

Reply:
A boolean is saved as a number, 0 or 1, so yes, you should be able to switch to a number or a string and use the 0 and 1 values. I didn't test this specific case, so I suggest to try it and make sure that it works. But I see that you added more options, so never mind.